### PR TITLE
[IMP] pylint: Enable check to verify duplicate id in ir.model.access.csv files

### PR DESCRIPTION
**Pull request by lescobarvx**
[IMP] pylint: Enable check to verify duplicate id in ir.model.access.csv files

### CODE REVIEW

File: tests/test_repo/broken_module/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,2 +1,5 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "access_test_model","access_test_model","model_test_model","base.group_user",1,1,1,1
+"access_test_model2","access_test_model2","model_test_model2","base.group_user",1,1,1,1
+"access_test_model2","access_test_model3","model_test_model3","base.group_user",1,1,1,1
+"access_test_model4","access_test_model4","model_test_model4","base.group_user",1,1,1,1
```
Comment (dreispt) [inline tests/test_repo/broken_module/security/ir.model.access.csv]: I think that the model used should be `model_test_model` in all lines.

**Comment by dreispt** [comment]
Don't you need to bump the expected errors counter?

File: tests/test_repo/broken_module/security/ir.model.access.csv (hunk 1 above)
Comment (moylop260) [inline tests/test_repo/broken_module/security/ir.model.access.csv]: This is a module broken... We can use a bad model name here

Comment (dreispt) [inline tests/test_repo/broken_module/security/ir.model.access.csv]: That will raise an error, but for the wrong reason (invalid model), thus not performing the check on the error we want. You're not testing the duplicate ID.

Comment (moylop260) [inline tests/test_repo/broken_module/security/ir.model.access.csv]: You are right!
(I don't saw the build result, my bad... fixing...)

**Comment by moylop260** [comment]
Expected errors counter increased, fix csv and rebased.

**Comment by dreispt** [comment]
LGTM :+1:

**Comment by nhomar** [comment]
:+1:

**Comment by moylop260** [comment]
Included from https://github.com/OCA/maintainer-quality-tools/commit/22a6a0bce39705942b112cd1e240ee00c1ecee9b

**Comment by dreispt** [comment]
I would rather prefer that we perform regular Merge: it makes clearer what was the PR that originated the commits, and we can then refer to it's discussion for more insights. 
IM(H)O fast forward merges are good for projects with a small well defined team (or one guy), not for repos with such a large and distributed responsibility.

**Comment by lasley** [comment]
They also break GPG verification from the original author & force the merger to deal with the commit note. Cool feature, but not really viable IMO

**Pull request closed without merging**